tests/test_edr.py

```diff
@@ -14,7 +14,7 @@
 import contextlib
 import numpy
 import pandas
-import panedr
+import panedrlite as panedr
 import re
 
 # On python 2, cStringIO is a faster version of StringIO. It may not be
```

Comment:
Can you fix these imports to be py3+ only now? The py2 code paths aren't necessary anymore.

Reply:
Done